test/integration/sanity/collection-variable.test.js

```diff
@@ -0,0 +1,117 @@
+describe('Collection Variables', function() {
+    var testrun;
+
+    before(function(done) {
+        this.run({
+            globals: {
+                values: [
+                    {key: 'global-var', value: 'global var value', name: 'global-var', enabled: true},
+                    {key: 'user', value: 'incorrect username', name: 'user', enabled: true}
+                ]
+            },
+            collection: {
+                variable: [
+                    {key: 'user', value: 'postman', enabled: true},
+                    {key: 'pass', value: 'incorrect password', enabled: true},
+                    {key: 'echo-url', value: 'https://postman-echo.com', enabled: true}
+                ],
+                item: {
+                    name: 'Collection Variables Test Request',
+                    event: [{
+                        listen: 'test',
+                        script: {
+                            exec: [`
+                                console.log('test', {
+                                    'global-var': pm.variables.get('global-var'),
```

Comment:
Could you update the tests to `pm.variables.toObject` once `postman-collection` is released?